mlserver/parallel/pool.py

```diff
@@ -191,12 +191,23 @@ async def unload_model(self, model: MLModel):
         )
 
     def _should_load_model(self, model: MLModel):
-        # NOTE: This is a remnant from the previous architecture for parallel
-        # workers, where each worker had its own pool.
-        # For backwards compatibility, we will respect when a model disables
-        # parallel inference.
-        if not model.settings.parallel_workers:
-            return False
+        if model.settings.parallel_workers is not None:
+            logger.warning(
+                "DEPRECATED!! The `parallel_workers` setting has now been moved "
```

Comment:
I guess we could look towards full removal in 1.2 and maybe add a note in the warning